app/src/main/res/values-night/color.xml

```diff
@@ -1,14 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="Transparent">#00000000</color>
 
-    <color name="Primary">#536dfe</color>
+    <color name="Primary_100">#DFE4F1</color>
```

Comment:
use only small litteral and underscope in .xml

Reply:
ok